recipes/r-tidytable/LICENSE

```diff
@@ -0,0 +1,21 @@
+# MIT License
```

Comment:
No need to include that file.

What is reverenced under `license_file:` is enough (it automatically combines the general MIT license file with the included details of the LICENSE file of the cran source tarball.

Reply:
Thanks @dbast!